### DISCUSSION TITLE
Do I need to purchase three URLS? Domain Name, Internal domain name, and short link domain name

### DISCUSSION THREAD

**Discussion by ybinstock** [mozilla/hubs#3257] [Hubs Cloud Questions] [answered]
I already purchased two of them through route 53 - virtualcelebrations.io, virtualcelebrations.link. Can my Internal domain name be the same as my domain name?

**Comment by robinkwilson** [comment, marked as answer]
Yes! You can deploy with only 2 domains.

You're technically following Recipe 1: deploying only using two domains: https://hubs.mozilla.com/docs/hubs-cloud-aws-domain-recipes.html#recipe-1-dedicated-domain-on-route-53
We follow Recipe 1 in the 3 part video series. Here's Part 2, deploying the stack: https://www.youtube.com/watch?v=W0erzlDpo0U

**Reply by ybinstock** [reply]
Thanks for all your help @robin-k-wilson!

Just launched the stack and waiting for it to go live. Fingers crossed.
